### ISSUE TITLE
Crashes in production

### ISSUE THREAD

**Issue by DanAndreasson** [Puigcerber/angular-country-picker#7]
I am receiving this error when trying to use it in production. In a rails stack on Heroku. I believe it is because of `uglifier` version `2.7.2`. Works like a charm in dev. 

```
application-2c947aab4f0e7d21ce429382427fd52330628d3a64dcc902288678ba9a990da1.js:59 Error: [$injector:unpr] Unknown provider: eProvider <- e <- pvpCountryPickerDirective
http://errors.angularjs.org/1.5.0/$injector/unpr?p0=eProvider%20%3C-%20e%20%3C-%20pvpCountryPickerDirective
    at application-2c947aab4f0e7d21ce429382427fd52330628d3a64dcc902288678ba9a990da1.js:58
    at application-2c947aab4f0e7d21ce429382427fd52330628d3a64dcc902288678ba9a990da1.js:58
    at Object.i [as get] (application-2c947aab4f0e7d21ce429382427fd52330628d3a64dcc902288678ba9a990da1.js:58)
    at application-2c947aab4f0e7d21ce429382427fd52330628d3a64dcc902288678ba9a990da1.js:58
    at i (application-2c947aab4f0e7d21ce429382427fd52330628d3a64dcc902288678ba9a990da1.js:58)
    at r (application-2c947aab4f0e7d21ce429382427fd52330628d3a64dcc902288678ba9a990da1.js:58)
    at Object.a [as invoke] (application-2c947aab4f0e7d21ce429382427fd52330628d3a64dcc902288678ba9a990da1.js:58)
    at application-2c947aab4f0e7d21ce429382427fd52330628d3a64dcc902288678ba9a990da1.js:58
    at o (application-2c947aab4f0e7d21ce429382427fd52330628d3a64dcc902288678ba9a990da1.js:58)
    at Object.<anonymous> (application-2c947aab4f0e7d21ce429382427fd52330628d3a64dcc902288678ba9a990da1.js:58)(anonymous function) @ application-2c947aab4f0e7d21ce429382427fd52330628d3a64dcc902288678ba9a990da1.js:59
```

**Comment by Puigcerber** [comment]
Hi @DanAndreasson! Which version of the module are you using? I believe it might be related to #6 and has been already fixed for version **2.0.1**.

**Comment by DanAndreasson** [comment]
I am running `1.1.1` for the moment. I'll update during the day and get back to you on how it goes. Thanks for the quick response!

**Comment by DanAndreasson** [comment]
@Puigcerber yep that solved it! Now running `2.0.1` without any issues! Cheers
